ts/Stock/StockToolsBindings.ts

```diff
@@ -348,9 +354,12 @@ bindingsUtils.manageIndicators = function (
             series.remove(false);
 
             if (indicatorsWithAxes.indexOf(series.type) >= 0) {
-                const removedYAxisHeight = yAxis.options.height;
+                const removedYAxisProps = {
+                    height: yAxis.options.height,
+                    top: yAxis.options.top
+                } as Highcharts.AxisPositionsObject;
                 yAxis.remove(false);
-                navigation.resizeYAxes(removedYAxisHeight as string);
+                navigation.resizeYAxes(removedYAxisProps);
```

Comment:
There is something wrong with `AxisPositionsObject` interface declaration if you still need to cast it here like this.